jaxb/pom.xml

```diff
@@ -54,14 +54,12 @@ for configuring data-binding.
       <artifactId>jackson-databind</artifactId>
     </dependency>
 
-    <!--  and actual JAXB annotations, types -->
-    <!-- NOTE! Despite groupId, this is "old" JAXB, not Jakarta
-         (3.x is real Jakarta one)
-      -->
+    <!-- Tolerate [2.2,3) for javax.xml.bind Import-Package in the generated MANIFEST.MF -->
+    <!-- https://mvnrepository.com/artifact/javax.xml.bind/jaxb-api -->
     <dependency>
-      <groupId>jakarta.xml.bind</groupId>
-      <artifactId>jakarta.xml.bind-api</artifactId>
-      <version>${version.jaxb.impl}</version>
+      <groupId>javax.xml.bind</groupId>
+      <artifactId>jaxb-api</artifactId>
+      <version>2.2.12</version>
```

Comment:
Shouldn't this instead change `version.jaxb.impl` property?

Reply:
No I don't think so as there is only a 2.3.3 or 2.3.2 available.

Reply:
Looks like `version.jaxb.impl` is used to get a JAXB impl for tests:

```
    <!-- may also need JAXB impl for tests -->
    <dependency>
      <groupId>org.glassfish.jaxb</groupId>
      <artifactId>jaxb-runtime</artifactId>
      <version>${version.jaxb.impl}</version>
      <scope>test</scope>
    </dependency>
```

Reply:
... but that does not seem to be needed, so I think I'll comment it out.